packages/astro/test/fixtures/styles-ssr/src/components/ReactModules.jsx

```diff
@@ -3,7 +3,7 @@ import Styles from './ReactModules.module.css';
 
 function ReactModules() {
   return (
-    <h1 id="react-modules" className={Styles.title}>
+    <h1 id="react-module-css" className={Styles.title}>
```

Comment:
All changes in this test file are arbitrary; just things like making the classnames between tests easier to distinguish from one another.

Barring insignificant changes, no tests were removed, but 12 new ones were added